src/command_modules/azure-cli-acs/HISTORY.rst

```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+2.2.3
++++++
+* Create role assignment to the subnet when it's provided.
+* Add new option "skip role assignment" for subnet when it's provided
+* Skip role assignment to subnet when assignment has already existed
```

Comment:
Nitpick: I would reword the last bullet point to say
`* Skip role assignment to subnet when assignment already exists`